moveit_ros/perception/depth_image_octomap_updater/src/depth_image_octomap_updater.cpp

```diff
@@ -210,27 +195,29 @@ bool host_is_big_endian()
 
 static const bool HOST_IS_BIG_ENDIAN = host_is_big_endian();
 
-void DepthImageOctomapUpdater::depthImageCallback(const sensor_msgs::ImageConstPtr& depth_msg,
-                                                  const sensor_msgs::CameraInfoConstPtr& info_msg)
+void DepthImageOctomapUpdater::depthImageCallback(const sensor_msgs::msg::Image::ConstSharedPtr& depth_msg,
+                                                  const sensor_msgs::msg::CameraInfo::ConstSharedPtr& info_msg)
 {
-  ROS_DEBUG_NAMED(LOGNAME, "Received a new depth image message (frame = '%s', encoding='%s')",
-                  depth_msg->header.frame_id.c_str(), depth_msg->encoding.c_str());
-  ros::WallTime start = ros::WallTime::now();
+  RCLCPP_DEBUG(LOGGER, "Received a new depth image message (frame = '%s', encoding='%s')",
+               depth_msg->header.frame_id.c_str(), depth_msg->encoding.c_str());
+  rclcpp::Time start = rclcpp::Clock(RCL_SYSTEM_TIME).now();
+  rclcpp::Clock steady_clock(RCL_STEADY_TIME);
```

Comment:
`RCL_ROS_TIME` or `RCL_STEADY_TIME`?

Reply:
The original code seems to only use ROS time so I think we could stick to `node_->now()` and `rclcpp::Time/rclcpp::Duration` for all of these. Or is there any specific reason to switch to system time or steady time?